lite/examples/object_detection/android/app/src/main/java/org/tensorflow/lite/examples/detection/LegacyCameraConnectionFragment.java

```diff
@@ -55,6 +55,8 @@
   private int layout;
   /** An {@link AutoFitTextureView} for camera preview. */
   private AutoFitTextureView textureView;
+  private SurfaceTexture availableSurfaceTexture = null;
+  
```

Comment:
nit: Can you please remove spaces here? Our internal CI would complain about these.